model/common/tests/common/interpolation/stencil_tests/test_interpolate_edge_field_to_half_levels_vp.py

```diff
@@ -30,19 +31,23 @@ def interpolate_edge_field_to_half_levels_vp_numpy(
     return interpolation_to_half_levels_vp
 
 
+@pytest.mark.embedded_only
 class TestInterpolateToHalfLevelsVp(test_helpers.StencilTest):
     PROGRAM = interpolate_edge_field_to_half_levels_vp
     OUTPUTS = ("interpolation_to_half_levels_vp",)
 
     @staticmethod
-    def reference(grid, wgtfac_e: np.ndarray, interpolant: np.ndarray, **kwargs) -> dict:
+    def reference(
+        wgtfac_e: np.ndarray,
```

Comment:
```suggestion
        grid: base_grid.Grid,
        wgtfac_e: np.ndarray,
```
grid is always injected as first argument, we should make it a keyword only argument. I believe I have a TODO somewhere, but needs refactoring of many tests.